pkgs/os-specific/linux/zfs/generic.nix

```diff
@@ -278,6 +281,7 @@ let
         ) innerArgs;
 
         inherit tests;
+        features = { inherit withSystemd; };
```

Comment:
This looks like yet another undocumented, ad-hoc thing. I'm not sure if that's a nice idea tbh.
Also related: https://github.com/NixOS/nixpkgs/pull/405551#discussion_r2083273869